tests/test_iso_tx_basic/src/main.xc

```diff
@@ -11,45 +9,37 @@
 #define EP_COUNT_IN    (6)
 #endif
 
-/* Endpoint type tables */
+#ifndef PKT_LENGTH_START
+#define PKT_LENGTH_START 	(10)
+#endif
+
+#ifndef PKT_LENGTH_END
+#define PKT_LENGTH_END 		(14)
+#endif
+
+#include "shared.h"
+
 XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL,
-                                                XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL};
+                                                XUD_EPTYPE_ISO,
```

Comment:
THis line appears to have a different level of indentation than the ones following.  Is that intentional?

Reply:
Fixed